src/services/credentialService.ts

```diff
@@ -15,30 +27,152 @@ import {
   IVerificationResult,
   IVerificationSubResult,
 } from '../types';
-
 import {removeCredentialBranding} from './brandingService';
 
+const logger: EventLogger = new EventLoggerBuilder()
+  .withContext(agentContext)
+  .withLogLevel(LogLevel.INFO)
+  .withSystem(System.CREDENTIALS)
+  .withSubSystem(SubSystem.VC_PERSISTENCE)
+  .withInitiatorType(InitiatorType.SYSTEM)
+  .build();
+
 export const getVerifiableCredentialsFromStorage = async (): Promise<Array<UniqueVerifiableCredential>> => {
+  await logger.logEvent({
+    type: LoggingEventType.AUDIT,
+    data: {
+      level: LogLevel.TRACE,
```

Comment:
Same issue as getContacts

Reply:
we do not have a way to see the difference